src/DockerSdk/DockerSdk.csproj

```diff
@@ -0,0 +1,21 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<PackageId>DockerSdk</PackageId>
+		<Description>DockerSdk is a library that allows you to interact with the Docker Remote API programmatically with fully asynchronous, non-blocking, and object-oriented code in your .NET applications.</Description>
+		<AssemblyName>DockerSdk</AssemblyName>
+		<EnableNETAnalyzers>true</EnableNETAnalyzers>
+	</PropertyGroup>
+
+	<ItemGroup>
```

Comment:
.editorConfig file is commited but removed from project here, is that what you want to do?

Reply:
I just didn't want to see it in Solution Explorer. It's still respected. It doesn't seem to effect whether it's respected.